nevergrad/optimization/optimizerlib.py

```diff
@@ -762,6 +763,84 @@ def _internal_provide_recommendation(self) -> ArrayLike:
         return self.avg
 
 
+@registry.register
+class SplitOptimizer(base.Optimizer):
+    """Combines optimizers, each of them working on their own variables.
+    
+    num_optims: number of optimizers
+    num_vars: number of variable per optimizer.
+
+    E.g. for 5 optimizers, each of them working on 2 variables, we can use:
+    opt = SplitOptimizer(instrumentation=10, num_workers=3, num_optims=5, num_vars=[2, 2, 2, 2, 2])
+    or equivalently:
+    opt = SplitOptimizer(instrumentation=10, num_workers=3, num_vars=[2, 2, 2, 2, 2])
+    Given that all optimizers have the same number of variables, we can also do:
+    opt = SplitOptimizer(instrumentation=10, num_workers=3, num_optims=5)
+
+    This is 5 parallel (by num_workers = 5).
+
+    Be careful! The variables refer to the deep representation used by optimizers.
+    For example, a categorical variable with 5 possible values becomes 5 continuous variables.
+    """
+
+    def __init__(self, instrumentation: Union[int, Instrumentation], budget: Optional[int] = None, num_workers: int = 1, num_optims: Optional[int] = None, num_vars: Optional[List[Any]] = None, default_multivariate_optimizer: Any = CMA, default_monovariate_optimizer: Any = RandomSearch) -> None:
```

Comment:
Call the optimizers without the default_ prefix. You only use the default if you don't set it.
I'm still a noob at forcing parameter types in python but can't you use base.Optimizer instead of Any for their types ?

Reply:
Right! Done.